apps/frontend/src/App.vue

```diff
@@ -8,7 +8,11 @@ import DefaultLayout from './layouts/default.vue'
   <div class="min-h-screen bg-white transition-all duration-300 ease-in-out dark:bg-gray-900">
     <Toaster position="top-right" :expand="true" :rich-colors="true" />
     <DefaultLayout>
-      <RouterView />
+      <template #default="{ changeTitle, setActions, setHidden, setCollapsed }">
+        <RouterView v-slot="{ Component }">
+          <component :is="Component" :change-title="changeTitle" :set-actions="setActions" :set-hidden="setHidden" :set-collapsed="setCollapsed" />
+        </RouterView>
+      </template>
```

Comment:
这里用 template 就好了，不需要额外写 slot，会自动处理的

```suggestion
      <RouterView />
```